app/views/helpers/inputRadioGroup.scala.html

```diff
@@ -0,0 +1,56 @@
+@*
```

Comment:
@Muhammad-Ali-Ahmad would you be able to give this a look?

Reply:
The markup should look like it is in design system: https://design-system.service.gov.uk/components/radios/
And for error state: https://design-system.service.gov.uk/components/error-message/

At the moment I cant see "form-group-error form-field--error" classes when there is an error. Does the radio options look as it should in error state? If not then these classes should be added to the <div class="form-group"> with condition that when there is error then add these classes.

Also there is no need to repeat H1. H1 can be removed from taxs_index.scala.html and inside legend, it should wrap the text in H1 and not have visually hidden class. 

Error message should have class "error-message".

Reply:
As discussed this should be handled as part of separate / Accessibility ticket .
Changes are fine as long as we are keeping the original behaviour intact.

Reply:
Can you get a ticket raised for this please

Reply:
DDCNL-2204 is created for the same.